### ISSUE TITLE
[Page list ] List Sort doen't work

### ISSUE THREAD

**Issue by Pandakls** [KleeGroup/focus-components#579]
### Context

I'm actually displaying a table list whith it's columns headers.
```javascript
    render(){
        const {currentParamId} = this.state;
        const columns = {
            libelle: {label: 'libelle', sort: 'asc'},
            description: {label: 'description', noSort: true},
            valeur: {label: 'valeur', noSort: true}};
        return (
            <Block title='administration.paramTech'>
                <SmartList ref='paramRefList' {...this._getListPageProps()} columns = {columns}/>
            </Block>
        );
    }
```
The line looks like this : 
```javascript
let Button = FocusCore.components.common.button.action.component;
const Icon = FocusComponents.common.icon.component;
const {column: Column, grid: Grid} = FocusComponents.common;

//Creates a line for the component.
module.exports = React.createClass({
    mixins: [FocusComponents.list.table.line.mixin],
    definitionPath: "parametreTechnique",
    _editHandler() {
        this.props.onLineClick(this.props.data);
    },
    renderLineContent: function(data){
        return (
            <tr className='hoverButton'>
                <td>{this.textFor("libelle", {})}</td>
                <td>{this.textFor("description", {})}</td>
                <td>{this.textFor("valeur", {})}</td>
                <td>
                    <button className='mdl-button mdl-js-button' onClick={this._editHandler}>
                        <Icon name='create'/>
                    </button>
                </td>
            </tr>
        );
    }
});
```
### Actual behaviour

I've got the error :
Invariant Violation: findFirstReactDOMImpl(...): Unexpected detached subtree found when traversing DOM from node.
when acting on the 'sort' button (little red arrow) :
![image](https://cloud.githubusercontent.com/assets/7700690/10607481/fada9520-7739-11e5-892a-77935a4ba1e6.png)
 "explode" my view (nothing left on screen) and throw this error :
![image](https://cloud.githubusercontent.com/assets/7700690/10607497/19b0aade-773a-11e5-9be9-8bbcd89fd8bc.png)
![image](https://cloud.githubusercontent.com/assets/7700690/10607500/20efbc72-773a-11e5-9707-662cf196ae59.png)

### Expected behaviour
I don't know what the sorting action is supposed to do between asking the server for a new sorted list or just sort what's in the storeList. But the error is not explicit.
If the list has only one page, the action should simply sort the list.

### Versions
Focus-core : 0.11.0
Focus-component: 0.7.3-3
Focus-File: 0.1.2

**Comment by Pandakls** [comment]
:+1:

**Comment by pierr** [comment]
@Pandakls @Jerom138 Could you check that this issue was fixed by #663 which was a duplicated of this issue according to me.
It was released in `FocusComponents v0.7.5`.
Please reopen the issue if it is not the case.
